crates/libs/windows/src/Windows/Win32/Graphics/Direct3D12/mod.rs

```diff
@@ -18269,17 +18269,29 @@ impl ID3D12CommandQueue {
     }
     #[doc = "*Required features: 'Win32_Graphics_Direct3D12', 'Win32_Foundation'*"]
     #[cfg(feature = "Win32_Foundation")]
-    pub unsafe fn UpdateTileMappings<'a, Param0: ::windows::core::IntoParam<'a, ID3D12Resource>, Param4: ::windows::core::IntoParam<'a, ID3D12Heap>>(&self, presource: Param0, numresourceregions: u32, presourceregionstartcoordinates: *const D3D12_TILED_RESOURCE_COORDINATE, presourceregionsizes: *const D3D12_TILE_REGION_SIZE, pheap: Param4, numranges: u32, prangeflags: *const D3D12_TILE_RANGE_FLAGS, pheaprangestartoffsets: *const u32, prangetilecounts: *const u32, flags: D3D12_TILE_MAPPING_FLAGS) {
-        (::windows::core::Interface::vtable(self).UpdateTileMappings)(::core::mem::transmute_copy(self), presource.into_param().abi(), ::core::mem::transmute(numresourceregions), ::core::mem::transmute(presourceregionstartcoordinates), ::core::mem::transmute(presourceregionsizes), pheap.into_param().abi(), ::core::mem::transmute(numranges), ::core::mem::transmute(prangeflags), ::core::mem::transmute(pheaprangestartoffsets), ::core::mem::transmute(prangetilecounts), ::core::mem::transmute(flags))
+    pub unsafe fn UpdateTileMappings<'a, Param0: ::windows::core::IntoParam<'a, ID3D12Resource>, Param4: ::windows::core::IntoParam<'a, ID3D12Heap>>(&self, presource: Param0, presourceregionstartcoordinates: &[D3D12_TILED_RESOURCE_COORDINATE], presourceregionsizes: &[D3D12_TILE_REGION_SIZE], pheap: Param4, prangeflags: &[D3D12_TILE_RANGE_FLAGS], pheaprangestartoffsets: &[u32], prangetilecounts: &[u32], flags: D3D12_TILE_MAPPING_FLAGS) {
+        (::windows::core::Interface::vtable(self).UpdateTileMappings)(
+            ::core::mem::transmute_copy(self),
+            presource.into_param().abi(),
+            presourceregionsizes.len() as _,
```

Comment:
presourceregionsizes is allowed to be null.

Reply:
`as_ptr_or_null` will pass a nullptr to the API if the slice is empty.